### DISCUSSION TITLE
예약 팁

### DISCUSSION THREAD

**Discussion by coldwarwithsoup** [kimytsc/covid-rest-vaccine-macro#40] [General]
타 매크로에 비해 이용자가 상대적으로 적어서 그런지 성공 후기가 없어서 저처럼 잘되는게 맞는지 전전긍긍하시는분 있을까봐 글 남깁니다.
제작자님 덕분에 어제 스크립트 돌리기 시작해서 오늘 오후에 접종하고 왔습니다.
우선 4가지 방법(네이버/카카오*크롬/크롬모바일) 모두 사용해봤는데
결과적으로는 예약성공한 2차례 모두 네이버/모바일 조합이었습니다.
다만 주의하실건 네이버의 경우 맵을 너무 크게 잡아서 입력하시면(병원리스트 2천개) 잔여백신 이용이 잠시동안 정지됩니다.

따라서 제작자님이 추천해주신 것처럼 대략 200~300여개 병원 리스트가 뜨도록 설정하시는게 가장 좋을 것 같습니다.
속도도 그게 더 빠른지 카카오는 800여개 리스트가 뜨도록 돌렸는데 오히려 네이버 200개가 먼저 잡히더라구요.
그래서 본인이 원하는 지역에 과하지 않게 적당한 크기로 맵을 잡으시는게 성공하는 팁이 될 것 같습니다.
또 네이버 예약창에서 "잔여백신이 없습니다" 라고 출력되는데 위쪽 시간 업데이트만 정상적으로 돌아가고 있다면 잔여백신이 생겼을때 정상적으로 예약이 됩니다. 너무 애타지 말고 차분히 기다리시면 될거 같습니다.

다시 한번 제작자님께 감사드리며, 간절히 바라면 스크립트가 이루어줍니다.

**Comment by worud47** [comment]
안녕하세요.
크롬 돌리면서 컴퓨터로 다른작업 해도 상관없을까요?

**Reply by kimytsc** [reply]
다른 작업을 해도 무관합니다.

다만 크롬 브라우저를 이용하실때는 예약 시도중인 창을 별도로 띄워두시는걸 추천드립니다. 브라우저가 퍼포먼스를 위해 비활성화된 창이나 탭은 스크립트를 느리게 동작시킬 수 있습니다.
